config/peerpods/podvm/redhat.repo

```diff
@@ -0,0 +1,6058 @@
+#
+# Certificate-Based Repositories
+# Managed by (rhsm) subscription-manager
+#
+# *** This file is auto-generated.  Changes made here will be over-written. ***
```

Comment:
Two questions here:

1. If the file is auto-generated, is there any way to generate it as part of the build? Probably not if it's based on `subscription-manager`, but asking just in case.
2. It looks like this overshot by a mile or two. Do we really need JBoss or RHEL AI repositories for the podvm builder?

If we don't need that many repositories, is there an inexpensive way to prune it and trim it down a little?

This seems particularly true given that I don't find `layered` once in the `rpms.lock.yaml` file, and that seems to be in the URLs for many of these repos. So that seems to indicate we don't actually use them.

Reply:
This is a copy of the repo file from the base image we build from, we've just edited it to have modifiable arch references. 
We probably could truncate it, but here I just followed the documentation from https://konflux.pages.redhat.com/docs/users/building/activation-keys-subscription.html#hermetic-network-isolated-builds
Note that it ends with:
```
Finally, commit the rpms.in.yaml, rpms.lock.yaml and redhat.repo to source control.
Konflux will use these files to prefetch RPMs for hermetic builds.
```

I think we could keep only the repos we enable. But cleaning it everytime we need to update it seems cumbersome :-/